presto-orc/src/main/java/com/facebook/presto/orc/metadata/DwrfMetadataReader.java

```diff
@@ -91,7 +91,8 @@ public PostScript readPostScript(byte[] data, int offset, int length)
         return new PostScript(
                 ImmutableList.of(),
                 postScript.getFooterLength(),
-                0,
+                postScript.getCacheSize(),
```

Comment:
what happens if postScript doesn't have the field? it's probably better to explicitly use optional and not rely on the default 0 value here

Reply:
It would return 0 same as other primitive fields.